src/main/java/org/folio/circulation/domain/validation/RequestLoanValidator.java

```diff
@@ -37,4 +51,31 @@ public RequestLoanValidator(LoanRepository loanRepository) {
         return singleValidationError(new ValidationError(message, parameters));
       }).map(loan -> requestAndRelatedRecords));
   }
+
+  public CompletableFuture<Result<RequestAndRelatedRecords>> refuseWhenUserHasAlreadyBeenLoanedAtLeastOneItem(
+    RequestAndRelatedRecords requestAndRelatedRecords) {
+
+    final Request request = requestAndRelatedRecords.getRequest();
+
+    CompletableFuture<Result<Collection<Item>>> itemsByInstanceId = itemByInstanceIdFinder.getItemsByInstanceId(UUID.fromString(request.getInstanceId()));
+    CompletableFuture<Result<MultipleRecords<Loan>>> openLoansByUserIdWithItem = loanRepository.findOpenLoansByUserIdWithItem(LOANS_PAGE_LIMIT, request.getUserId());
```

Comment:
I don't think we need these variables, I would suggest using these expressions directly in the pipeline below

Reply:
done